projects/1.16/assets/resourceful-bees/resourcefulbees/lang/zh_cn.json

```
@@ -1,442 +1,827 @@
-  "advancements.resourcefulbees.oreo_bee.title": "别吃它",
-  "advancements.resourcefulbees.oreo_bee.description": "找到并捕获罕见的奥利奥蜜蜂",
-  "advancements.resourcefulbees.root.title": "丰饶蜜蜂",
-  "advancements.resourcefulbees.scraper.title": "蜜脾收获装置",
-  "advancements.resourcefulbees.scraper.description": "获得刮刀",
-  "advancements.resourcefulbees.smoker.title": "蜜蜂安全入门指南",
-  "advancements.resourcefulbees.smoker.description": "获得烟熏器",
-  "advancements.resourcefulbees.t1_apiary.title": "温馨巢居",
-  "advancements.resourcefulbees.t1_apiary.description": "获得1级蜂场",
-  "advancements.resourcefulbees.t1_beehive.title": "蜂之窝棚",
-  "advancements.resourcefulbees.t1_beehive.description": "获得1级蜂巢",
-  "advancements.resourcefulbees.t1_hive_upgrade.title": "迷你蜂之窝棚",
-  "advancements.resourcefulbees.t1_hive_upgrade.description": "获得1级蜂巢升级",
-  "advancements.resourcefulbees.t2_apiary.title": "关爱蜜蜂",
-  "advancements.resourcefulbees.t2_apiary.description": "获得2级蜂场",
-  "advancements.resourcefulbees.t2_beehive.title": "蜜蜂公寓",
-  "advancements.resourcefulbees.t2_beehive.description": "获得2级蜂巢",
-  "advancements.resourcefulbees.t2_hive_upgrade.title": "小蜜蜂公寓",
-  "advancements.resourcefulbees.t2_hive_upgrade.description": "获得2级蜂巢升级",
-  "advancements.resourcefulbees.t3_apiary.title": "蜂杀战局",
-  "advancements.resourcefulbees.t3_apiary.description": "获得3级蜂场",
-  "advancements.resourcefulbees.t3_beehive.title": "蜜蜂房屋",
-  "advancements.resourcefulbees.t3_beehive.description": "获得3级蜂巢",
-  "advancements.resourcefulbees.t3_hive_upgrade.title": "迷你蜂之房屋",
-  "advancements.resourcefulbees.t3_hive_upgrade.description": "获得3级蜂巢升级",
-  "advancements.resourcefulbees.t4_apiary.title": "该结束了",
-  "advancements.resourcefulbees.t4_apiary.description": "获得4级蜂场",
-  "advancements.resourcefulbees.t4_beehive.title": "蜜蜂大厦",
-  "advancements.resourcefulbees.t4_beehive.description": "获得4级蜂巢",
-  "advancements.resourcefulbees.t4_hive_upgrade.title": "小蜜蜂大厦",
-  "advancements.resourcefulbees.t4_hive_upgrade.description": "获得4级蜂巢升级",
-  "advancements.resourcefulbees.wax.title": "上蜡刮蜡，持之以恒",
-  "advancements.resourcefulbees.wax.description": "获得蜜蜡"
-}
\ No newline at end of file
+	"itemGroup.resourcefulbees": "缤纷蜜蜂",
+
+	"resourcefulbees.information.bee_type.vanilla": "蜜蜂类型：原版",
+	"resourcefulbees.information.bee_type.unknown": "蜜蜂类型：未知",
+	"resourcefulbees.information.bee_type.custom": "蜜蜂类型：",
+	"resourcefulbees.information.unknown_type": "未知类型",
+	"resourcefulbees.shift_info": "按住SHIFT以查看信息",
+	"resourcefulbees.ctrl_info": "按住CTRL以查看多方块结构信息",
+	"gui.resourcefulbees.yes": "是",
+	"gui.resourcefulbees.no": "否",
+	"gui.resourcefulbees.light.any": "任意",
+	"gui.resourcefulbees.light.day": "白天",
+	"gui.resourcefulbees.light.night": "夜晚",
+
+
+	"item.resourcefulbees.bee_box": "蜂盒",
+	"item.resourcefulbees.crafting_bee_box": "迷失的蜂盒",
+	"item.resourcefulbees.information.bee_box.temp_info": "包含迷失的蜜蜂。潜行时即可释放它们。（使用后消失）",
+	"item.resourcefulbees.information.bee_box.info": "最大可容纳10只蜜蜂。潜行时即可释放它们。",

```

```suggestion
	"item.resourcefulbees.information.bee_box.temp_info": "包含迷失的蜜蜂。潜行时使用即可释放它们。（使用后消失）",
	"item.resourcefulbees.information.bee_box.info": "最大可容纳10只蜜蜂。潜行时使用即可释放它们。",
```
原文：Use while sneaking to release them.